h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -156,6 +188,16 @@ class SearchBarController extends Controller {
               querySelector('[data-ref="searchBarDropdownItemTitle"]').
               innerHTML.trim();
```

Comment:
for readibility, it would be better to create two variables, the `titleEl` and then the `facet` so that it is not a 4 line assignment